topics/sql/index.md

```diff
@@ -0,0 +1,11 @@
+---
+created_by: Donald D. Chamberlin, Raymond F. Boyce
+display_name: SQL
+logo: sql.png
+released: 1986
+short_description: SQL is a standard language for storing, retrieving and manipulating data in databases.
+topic: sql
+related: database
+wikipedia_url: https://en.wikipedia.org/wiki/SQL
+---
+SQL stands for structured query language. It uses commands such as "select", "insert", "delete". Some common relational database management systems that use SQL are: Oracle, Sybase, Microsoft SQL Server, Access, Ingres, etc 
```

Comment:
`etc.`.

How about: `Oracle, MySQL, Microsoft SQL Server, PostgreSQL`? I think those are the most popular bunch.